lib/utils.js

```diff
@@ -733,7 +733,11 @@ exports.stackTraceFilter = function() {
       }
 
       // Clean up cwd(absolute)
-      list.push(line.replace(cwd, ''));
+      if (/\(?.+:\d+:\d+\)?$/.test(line)) {
```

Comment:
that's actually not a capture group, those are escaped parenthesis. your suggestion would only work for stack lines without parenthesis around the file+position info, but according to the `test/utils.js` both seem to exist.